src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+import duke.util.Parser;
+
+/**
+ * This is the main class of the program.
+ */
 public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Initialises the Duke object.
```

Comment:
If I'm not wrong, the CS2103T coding standard includes a line break between the JavaDoc description and parameter tags!

Reply:
Thanks for pointing it out, will get it fixed!